src/scripts/pokedex/PokedexHelper.ts

```diff
@@ -1,7 +1,7 @@
 import TypeColor = GameConstants.TypeColor;
 
 class PokedexHelper {
-    public static toggleStatisticShiny = ko.observable(true);
+    public static toggleStatisticShiny = ko.observable(false);
```

Comment:
Wouldn't this make it show the non-shiny image on the individual Pokémon modal?

Reply:
Oh god, my head is exploding this days, sorry! I will push the right tested things